### PR TITLE
Readme game instructions

### PR DESCRIPTION
**Pull request by martinnajjar12**
### Game Instructions

We have:

- Created a README.md file,
- Clarified how to start the game
- And showed the win cases.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,93 @@
-# tic-tac-toe
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+[![Run on Repl.it](https://repl.it/badge/github/edxco/tic-tac-toe)](https://repl.it/github/edxco/tic-tac-toe)
+# Tic-Tac-Toe
+
+> The famous game: Tic-Tac-Toe
+
+Additional description about the project and its features.
```
Comment (maelfosso) [inline README.md]: Kindly fully and clearly describe the TIC-TAC-TOE game

File: README.md [hunk 2]
```diff
@@ -1 +1,93 @@
...
+## Live Demo
+
+[Live Demo Link](https://repl.it/github/edxco/tic-tac-toe)
+
+
+## Getting Started
+ You need 2 players to play the game!
+
+
+To get a local copy up and running follow these simple example steps.
```
Comment (maelfosso) [inline README.md]: Please, add the command to get the local copy
```rb
git clone <repo_url>
```
What is the command the user should type to run it? Kindly add all the needed commands.

File: README.md [hunk 3]
```diff
@@ -1 +1,93 @@
...
+
+![screenshot](https://github.com/edxco/tic-tac-toe/blob/readme_game_instructions/images/modi5.jpg)
+
+Once the first player choose an index, that index will be changed to his sign as shown below:
+
+![screenshot](https://github.com/edxco/tic-tac-toe/blob/readme_game_instructions/images/modi4.jpg)
+
+Then it's other player turn. The second player now should choose his sign in order not to allow the first player to win.
+
+### Win cases
```
Comment (maelfosso) [inline README.md:54]: There are 9 winning cases. In addition to the job done so far, a description of what is a winning case.

File: README.md [hunk 4]
```diff
@@ -1 +1,93 @@
...
+![screenshot](https://github.com/edxco/tic-tac-toe/blob/readme_game_instructions/images/modi3.jpg)
+- Column line
+
+![screenshot](https://github.com/edxco/tic-tac-toe/blob/readme_game_instructions/images/modi2.jpg)
+- Row line
+
+![screenshot](https://github.com/edxco/tic-tac-toe/blob/readme_game_instructions/images/modi1.jpg)
+
+### Run tests
+You can use RSpec and Rubocop to test the code.
```
Comment (maelfosso) [inline README.md]: What is the command to run? Add it, please, with also all the requirements if any.

**Review by maelfosso — CHANGES_REQUESTED** [review]
# Changes required :red_circle: 

Hello,

Thank you for the job done so far, 
However, there are some section that needs to be improved

Kindly the comments left below to know more about that and once done, please, submit a new code review request

Thanks

Mael FOSSO

**Review by expjazz — APPROVED** [review]
## Status: Project Approved :clap: 

You did a great job. Your readme is very professional :1st_place_medal: 

Good Luck and Happy Coding :pray:

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:_

**Pull request merged**